api/src/main/java/com/messagebird/objects/conversations/ConversationWebhookBaseRequest.java

```diff
@@ -0,0 +1,41 @@
+package com.messagebird.objects.conversations;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Contains common fields for webhook requests.
+ */
+public abstract class ConversationWebhookBaseRequest {
+    protected String url;
+    protected List<ConversationWebhookEvent> events;
+
+    public String getUrl() {
+        return url;
+    }
+
+    public void setUrl(String url) {
+        this.url = url;
+    }
+
+    public List<ConversationWebhookEvent> getEvents() {
+        return events;
+    }
+
+    public void setEvents(List<ConversationWebhookEvent> events) {
+        this.events = events;
+    }
+
+    protected abstract String getRequestName();
```

Comment:
This is not used anywhere, is it? In that case I propose removing it

Reply:
It is used in the ToString of the base.

Reply:
Ah ok, missed that 👍